lib_expense/src/test/java/com/example/expenselogger/lib_expense/domain/usecase/GetExpensesTest.kt

```diff
@@ -1,36 +1,23 @@
 package com.example.expenselogger.lib_expense.domain.usecase
 
 import com.example.expenselogger.lib_expense.domain.contract.ExpenseContract
-import com.example.expenselogger.lib_expense.domain.fakes.FakeExpenseContract
 import com.example.expenselogger.lib_expense.domain.model.DummyData
 import com.example.expenselogger_test_utils.TestAsyncExecutor
 import com.google.common.truth.Truth.assertThat
 import kotlinx.coroutines.test.runBlockingTest
 import org.junit.Test
+import org.mockito.Mockito.`when`
 import org.mockito.Mockito.mock
-import org.mockito.Mockito.verify
 
 public class GetExpensesTest {
 
-    private val expenseContract = FakeExpenseContract()
-    private val getExpenses = GetExpenses(
-        expenseContract,
-        TestAsyncExecutor()
-    )
+    private val expenseContract = mock(ExpenseContract::class.java)
+    private val getExpenses = GetExpenses(expenseContract, TestAsyncExecutor())
 
     @Test
-    public fun `verify that expenses returns a list of expenses`(): Unit = runBlockingTest {
-        val expense = DummyData.expense
-        expenseContract.saveExpense(expense)
+    public fun `verify that getExpenses usecase gets a list of expenses`(): Unit = runBlockingTest {
+        `when`(expenseContract.getExpenses()).thenReturn(listOf(DummyData.expense))
```

Comment:
Assert that the lists are the same, and not that the size is 1. What if the size is 1 and it contains the wrong data?

Reply:
Yea that's true. I will modify the test